packages/particle-cli/app/storybook/main.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')frontendFramework
+
+const APP_COMPONENT_LIBRARY = path.resolve(__dirname, '../../src/default')
```

Comment:
See directory comment above. This should resolve from a generated configuration file in the base of the project.